ci/appveyor/make_zip.bat

```diff
@@ -0,0 +1,62 @@
+mkdir C:\aditof-sdk\
+pushd C:\aditof-sdk
+
+if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" (
+    set folder=vs14
+) else if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" (
+    set folder=vs15
+)
+
+mkdir x64
```

Comment:
Doesn't make more sense to directly create all dir paths without all the pushing and popping? It would make these instructions more readable.
E.g.
`mkdir x64`
`mkdir x64\%folder%`
`mkdir x64\%folder%\bin`
`mkdir x64\%folder%\lib`
`etc.`